src/wallet.rs

```diff
@@ -86,6 +86,10 @@ impl Wallet {
         self.pubkey_bin.to_b58()
     }
 
+    pub fn address_as_vec(&self) -> Vec<u8> {
```

Comment:
Can you not do this? it's used in just one place in this PR and so not really needed

Reply:
I think the code reads better with it but OK